ibis/expr/timecontext.py

```diff
@@ -127,6 +127,40 @@ def canonicalize_context(
     return begin, end
 
 
+def construct_multi_index_series(
```

Comment:
can we have a more informative name for this. yes its constructing a MI series, but its doing it for time_context.
maybe

construct_time_context_aware_series

Reply:
+1

Reply:
Good idea, I changed the name to be `construct_time_context_aware_series`